dir.targets

```diff
@@ -110,4 +110,15 @@
     <Message Importance="High" Condition="'$(ConfigurationErrorMsg)' != ''" Text="$(MSBuildProjectFullPath) ConfigurationErrorMessage: $(ConfigurationErrorMsg)" />
   </Target>
 
+  <Target Name="GenerateReferenceAssembly">
+    <PropertyGroup>
```

Comment:
For reference I do this in the standard repo at https://github.com/dotnet/standard/blob/master/netstandard/src/netstandard.csproj#L45 we should share the common pattern.